package.json

```diff
@@ -134,7 +134,7 @@
     "start": "npm-run-all git:info start:server",
     "start:client": "node $NODE_DEBUG_OPTION build/src/start.client.js --open=true",
     "start:server": "node $NODE_DEBUG_OPTION --trace-warnings --expose_gc --optimize_for_size --always_compact --max_old_space_size=256 build/src/start.client.js",
-    "test": "mocha --require espower-typescript/guess --exit 'test/{*.ts,!(api|benchmark|bitbucket-api)/**/*.ts}'",
+    "test": "mocha --require espower-typescript/guess --exit 'test/{*.ts,!(api|benchmark)/**/*.ts}'",
     "test:api": "mocha --require espower-typescript/guess --exit 'test/api/**/*.ts'",
     "test:bitbucket-api": "mocha --require espower-typescript/guess --exit 'test/bitbucket-api/**/*.ts'",
```

Comment:
remove this one if it's going to be included in the regular tests?
... although this means people will need to set ATLASSIAN_USER and PASSWORD in order to run tests locally :-(

Reply:
I do think we should leave this separate; @ddgenome and I worked at not requiring collaborators on this project to do a bunch of credential setup in order to `npm run test`